### PR TITLE
Add an optional ramcache setting to volume.config to disable it

### PR DESCRIPTION
**Pull request by ezelkow1**
This adds an additional option of `ramcache=true/false` to the volume.config.
The default is true, so enabled, to leave current functionality as is and for
the general use case. If set to false then this will stop the cache from initializing
a ramcache for that specific volume and avoid the ramcache all together. This may
be desirable for people running with volumes composed of ram disks to avoid double
ram caching and reclaim memory+time

### CODE REVIEW

**Comment by ezelkow1** [comment]
apparently vscode might remove the word 'be' when auto-prettying? that or I have some strange consistency in typos

File: iocore/cache/CacheHosting.cc [hunk 1]
```diff
@@ -656,6 +656,7 @@ ConfigVolumes::BuildListFromString(char *config_file_path, char *file_buf)
     CacheType scheme  = CACHE_NONE_TYPE;
     int size          = 0;
     int in_percent    = 0;
+    bool ramcache     = true;
```
Comment (SolidWallOfCode) [inline iocore/cache/CacheHosting.cc:659]: I'd really prefer a slightly different name, to better distinguish this from "ram_cache", i.e. member that is an actual data structure and not a flag. I use the suffix "\_p" for "predicate" in such circumstances, although a prefix of "f_" is used in various places. Or "ramcache_enabled". Then you have things like
```
if (gvol[i]->cache_vol->ramcache_p) {
```
or
```
if (gvol[i]->cache_vol->ramcache_enabled) {
```

Comment (ezelkow1) [reply iocore/cache/CacheHosting.cc]: Agreed, renamed all those to ramcache_enabled

**Comment by ezelkow1** [comment]
> Looks good except a bit better naming to avoid future confusion.

Renamed all the ramcache booleans to ramcache_enabled

**Comment by zwoop** [comment]
Cherry-picked to v9.0.x branch.

**Pull request merged**